### PR TITLE
Reverse proxy: integrate front end project

### PR DESCRIPTION
**Pull request by nzcodarnoc**
This project puts it all together. It's more of a demo of how we could run the different projects in a larger orchestration system. Instruction on how to run are in the README.

It should show a mouth that says "hello" on page load. Then, whatever is typed into the chat widget, will be said by the mouth.

### CODE REVIEW

**Comment by JacOng17** [comment]
Hmmm, when running `docker-compose build && docker-compose up` I get this error:
image

I referred to: https://docs.docker.com/compose/compose-file/ & seems to be my Docker version is compatible.

My docker version: Docker version 19.03.6, build 369ce74a3c
My docker-compose version: docker-compose version 1.17.1, build unknown

File: docker-compose.yml [hunk 1]
```diff
@@ -1,10 +1,19 @@
 version: '3.3'
```
Comment (nzcodarnoc) [inline docker-compose.yml:1]: @JacOng17 I changed the version to one that is (hopefully) supported for you. I tested it my end and it build OK.

Comment (JacOng17) [reply docker-compose.yml:1]: Nice, yup. I changed it on my local & building now :+1:

**Review by JacOng17 — APPROVED** [review]
LGTM, docker-compose builts with version 3.3 but not 3.6. When building, pulls from the corresponding repos. :tada: :smiley: :+1:

File: docker-compose.yml (hunk 1 above)
Comment (JacOng17) [inline docker-compose.yml:1]:
```suggestion
version: '3.5'
```

Comment (JacOng17) [reply docker-compose.yml:1]: **3.5** works for me, just not **3.6** :frowning_face:

Comment (nzcodarnoc) [reply docker-compose.yml:1]: I'm gonna leave it at 3.3 now. I bumped it up to try and solve a problem with .env in the front-end project. Which it didn't.

**Pull request merged**